pkgs/by-name/cr/create-dmg/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  pname = "create-dmg";
+  version = "v1.2.2";
+
+  src = fetchgit {
+    url = "https://github.com/create-dmg/create-dmg";
+    hash = "sha256-oWrQT9nuFcJRwwXd5q4IqhG7M77aaazBG0+JSHAzPvw=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out
+    /usr/bin/make DESTDIR=$out prefix="" install
+  '';
```

Comment:
`InstallPhase` is not necessary in this case. This should yield the same result:
```
installFlags = [ "PREFIX=$(out)" ];
```